### ISSUE TITLE
Add thumbnail information to Vimeo Video object

### ISSUE THREAD

**Issue by ed-george** [ed-george/AndroidVimeoExtractor#2]
Add video thumbnail information to `VimeoVideo` object.

The API call returns it in a format similar (but not always exactly) to the following:

``` json
"thumbs": {
      "640": "https://i.vimeocdn.com/video/xxxx_640.jpg",
      "960": "https://i.vimeocdn.com/video/xxxx_960.jpg",
      "1280": "https://i.vimeocdn.com/video/xxxx_1280.jpg",
      "base": "https://i.vimeocdn.com/video/xxxx"
} 
```

**Comment by Jagruttam** [comment]
I am looking for the exact.
Please do it ASAP.

> Thanks in advance!

**Comment by ed-george** [comment]
@Jagruttam thanks for your comment - I have made the associated changes in the development branch ready for testing. 

In the meantime, feel free to use the following dependancy in your app level `build.gradle` file to get the changes immediately before I release a new version.

```
dependencies {
    compile 'com.github.ed-george:AndroidVimeoExtractor:development-SNAPSHOT'
}
```

If you have any further suggestions, please let me know or file a pull request.

**Comment by ed-george** [comment]
Fixed in version 1.1

**Comment by Jagruttam** [comment]
Thanks!
